src/DynamoCore/DSEngine/LiveRunnerServices.cs

```diff
@@ -53,7 +53,7 @@ public RuntimeMirror GetMirror(string var, bool verboseLogging)
             var mirror = liveRunner.InspectNodeValue(var);
 
             if (verboseLogging)
-                Log("LRS.GetMirror var: " + var + " " + (mirror != null ? mirror.GetStringData() : "null"));
+                Log(Properties.Resources.LRSGetMirror +" " + var + " " + (mirror != null ? mirror.GetStringData() : "null"));
```

Comment:
This can be improved as:

```csharp
var data = (mirror != null ? mirror.GetStringData() : "null");
Log(string.Format(Properties.Resources.LRSGetMirror, var, data));
```